p2p/node.go

```diff
@@ -185,8 +186,9 @@ func New(config Config) (*Node, error) {
 	routingDiscovery := discovery.NewRoutingDiscovery(kadDHT)
 
 	// Set up pubsub.
-	// TODO: Replace with WeijieSub. Using FloodSub for now.
-	ps, err := pubsub.NewFloodsubWithProtocols(nodeCtx, basicHost, []protocol.ID{pubsubProtocolID})
+	pubsub.GossipSubHistoryGossip = 10
+	pubsub.GossipSubHistoryLength = 20
```

Comment:
Can you add comments explaining what these configurations mean?

Reply:
They affect the size of the internal message cache used by GossipSub.

I actually just decided to remove them. The idea was that since we are sharing old messages, we should have a larger message cache. But after I wrote this code I realized old messages get a new message ID so increasing the size of the message cache doesn't help us at all.